rocky/katalogus/templates/boefjes.html

```diff
@@ -16,14 +16,19 @@
                     <h2>Boefjes</h2>
                     <p>
                         {% blocktranslate trimmed %}
-                            Boefjes gather factual information, such as by calling an
-                            external scanning tool like nmap or using a database like shodan.
+                            Boefjes are used to scan for objects. They detect vulnerabilities,
+                            security issues, and give insight. Each boefje is a separate scan that
+                            can run on a selection of objects.
                         {% endblocktranslate %}
                     </p>
                     <p class="emphasized">
                         <strong>{{ object_list|length }}</strong> Boefje{{ object_list|pluralize:"s" }} {% translate "available" %}
                     </p>
                 </div>
+                <div class="horizontal-view toolbar">
+                    <a class="button ghost"
+                       href="{% url "boefje_setup" organization_code=organization.code %}"><span class="icon ti-plus"></span>{% translate "Add Boefje" %}</a>
```

Comment:
This icon should get an `aria-label` or `aria-hidden="true"` - probably the latter if the link already has accessible text and the icon is purely decorative.